### PR TITLE
[JENKINS-66677] Localize "This folder is empty" text

### PR DESCRIPTION
**Pull request by FilipRazek**
See [JENKINS-66677](https://issues.jenkins.io/browse/JENKINS-66677).
Also adds minor French translations for text encountered while using the interface to check that the change appeared correctly.
The issue [JENKINS-66680](https://issues.jenkins.io/browse/JENKINS-66680) can probably also be closed, since the sidebar seems to be fully translated.

### Testing done
image

### Proposed changelog entries

N/A

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

File: core/src/main/resources/hudson/model/AllView/noJob_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/model/AllView/noJob_fr.properties:31]:
```suggestion
thisFolderIsEmpty=Ce dossier est vide

```
Missing EOL

File: core/src/main/resources/hudson/model/Job/configure_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/model/Job/configure_fr.properties:30]:
```suggestion
Configure=Configurer

```

File: core/src/main/resources/hudson/model/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/model/Messages_fr.properties:117]:
```suggestion
  Job legacy polyvalent qui récupère l''état depuis un outil de gestion de version au plus, exécute les étapes de build en série, \
```
Optional suggestion, more close to other existing translations of "job", "legacy" and "SCM"

**Review by lemeurherve — COMMENTED** [review]
Some small suggestions, thanks for the pull request!

**Review by lemeurherve — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Thanks!

---

/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
